src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,130 @@
+import { useEffect, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { getPeople } from '../api';
+import { Loader } from './Loader';
+
+export const PeoplePage = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [isError, setIsError] = useState<boolean>(false);
+  const { slugName } = useParams();
+
+  useEffect(() => {
+    setLoading(true);
+    getPeople()
+      .then((people) => {
+        setPeopleFromServer(people);
+        setLoading(false);
```

Comment:
Not fixed
![2023-10-16_15h39_58](https://github.com/mate-academy/react_people-table-basics/assets/61904995/d4482833-03b7-4242-af18-d4a328b245c0)

```suggestion

```